templates/actors/group/tabs.hbs

```diff
@@ -0,0 +1,9 @@
+<nav data-group="primary">
```

Comment:
Is this any different than the `systems/dnd5e/templates/shared/sidebar-tabs.hbs` we use for the other sheets?